src/commands/service/create.md

```diff
@@ -0,0 +1,25 @@
+## NOTE
```

Comment:
Seems like we should at least keep the Description portion of this command, even if it's a repeat from what's output by `spk service create -h`, just so things are consistent.

Reply:
ok